### PR TITLE
minor changes to bring docs up to date with new config

### PR DESCRIPTION
**Pull request by ChristinaLK**
- adding login node info to hpc overview
- deleting note about new cluster
- changing default cpu request to 20 cores per node

### CODE REVIEW

File: hpc-job-submission.md [hunk 1]
```diff
@@ -43,10 +43,10 @@ and 64GB of memory each (so 32 cores and 128 GB of memory total), on the
 ``` {.sub}
 #!/bin/sh
 #This file is called submit-script.sh
-#SBATCH --partition=univ        # default "univ", if not specified
+#SBATCH --partition=univ2       # default "univ", if not specified
```
Comment (jmvera255) [inline hpc-job-submission.md:46]: comment still needs to be changed 
`#default "univ2"`

**Review by jmvera255 — CHANGES_REQUESTED** [review]
one minor change requested

File: hpc-job-submission.md (hunk 1 above)
Comment (jmvera255) [inline hpc-job-submission.md:46]:
```suggestion
#SBATCH --partition=univ2       # default "univ2", if not specified
```

**Review by jmvera255 — DISMISSED** [review]

**Review by jmvera255 — APPROVED** [review]
@ChristinaLK I caught a few more `univ` specifics that I changed to `univ2` in additional commits

**Comment by ChristinaLK** [comment]
perfect, thanks.

**Pull request merged**